packages/react-native-reanimated/plugin/src/plugin.ts

```diff
@@ -50,6 +52,14 @@ module.exports = function (): PluginItem {
           });
         },
       },
+      Program: {
+        enter(_path, state) {
+          runWithTaggedExceptions(() => {
+            // Reset worklet number.
+            state.workletNumber = 1;
```

Comment:
Let's use worklet hash instead as it is more stable than autoincrement.

Reply:
Worklet hashes depend on the function name 🤓

Reply:
Please do anyway

Reply:
TBD